### DISCUSSION TITLE
What happens when after a download has completed with an error, the user restarts the download of the same playlist to the same folder?

### DISCUSSION THREAD

**Discussion by Korb** [ErrorFlynn/ytdlp-interface#166] [Q&A]
The question is, does the download attempt to continue, or does the app start the download from the beginning regardless of what was downloaded before?

**Comment by ErrorFlynn** [comment]
Sorry for the late reply, I just noticed this now. If you haven't deleted the `.part` files, yt-dlp will resume the download. Otherwise it will start from the beginning.
